src/opentslm/time_series_datasets/mimiciv_ecg/mimiciv_ecg_loader.py

```
@@ -0,0 +1,189 @@
+    # Create samples from ECG files
+    all_samples = []
+    for ecg_file in tqdm(ecg_files, desc="Creating samples"):
+        try:
+            ecg_id = extract_ecg_id(ecg_file)
+            sample = {
+                'ecg_id': [ecg_id],
+                'ecg_paths': [ecg_file + '.dat'],
+                'sample_id': len(all_samples),
+            }
+            all_samples.append(sample)
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)